browser/js/reviews/review-item/review-item.js

```diff
@@ -0,0 +1,9 @@
+app.directive('HfReviewItem', function(reviewFactory) {
```

Comment:
also, what is reviewFactoy? did you mean ReviewsFactory? I don't know directives well so not sure what this should be.